src/com/palmergames/bukkit/TownyChat/Command/ChannelCommand.java

```diff
@@ -388,4 +391,22 @@ public static void parseChannelJoin(Player player, String[] split) {
 
 		TownyMessaging.sendMessage(player, Translation.of("tc_you_joined_channel", chan.getName()));
 	}
+
+	public static void parseChannelSoundToggle(Player player, String[] split) {
+		if (split.length == 0) {
+			for (String line : channel_help) {
+				player.sendMessage(line);
+			}
+			return;
+		}
+		Channel chan = plugin.getChannelsHandler().getChannel(split[0]);

```

Comment:
This chan needs to be vetted as being not-null, otherwise an NPE is thrown on line 404 when a random word is used instead of a channel name.